src/components/application_manager/src/commands/mobile/alert_maneuver_request.cc

```diff
@@ -238,6 +238,25 @@ bool AlertManeuverRequest::IsWhiteSpaceExist() {
       }
     }
   }
+  if ((*message_)[strings::msg_params].keyExists(strings::soft_buttons)) {
+    const smart_objects::SmartArray* sb_array =
```

Comment:
@okozlovlux check this pointer with DCHECK (soft buttons should be array, if it is not - our API parser works not correctly)